core/src/main/scala/com/pingcap/tispark/v2/TiDBTableScanBuilder.scala

```diff
@@ -0,0 +1,14 @@
+package com.pingcap.tispark.v2
+
+import com.pingcap.tispark.TiTableReference
+import org.apache.spark.sql.connector.read.{Scan, ScanBuilder}
+import org.apache.spark.sql.types.StructType
+
+class TiDBTableScanBuilder(
```

Comment:
use case class. `JDBCScanBuilder` is also case class

Reply:
It has been modified. thx.